frontend/pages/DashboardPage/cards/ActivityFeed/ActivityItem/ActivityItem.tsx

```diff
@@ -568,6 +587,12 @@ const getDetail = (
     case ActivityType.TransferredHosts: {
       return TAGGED_TEMPLATES.transferredHosts(activity);
     }
+    case ActivityType.EnableWindowsMdm: {
+      return TAGGED_TEMPLATES.enableWindowsMdm(activity);
+    }
+    case ActivityType.DisableWindowsMdm: {
+      return TAGGED_TEMPLATES.disableWindowsMdm(activity);
```

Comment:
Mentioning in case the `TAGGED_TEMPLATES` need to have the same name as the activity (probably best), they'd need to be changed too to `enabledWindowsMdm` and `disabledWindowsMdm`.